### PR TITLE
Fix `@font-face` overrides

### LINKED ISSUE #24850: Issue title bold becomes weight
### Description

Issue title becomes too bold
图片

Maybe caused by #24827

### Gitea Version

#24827

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

local compiling

### Database

None

### PR DESCRIPTION
**Pull request by silverwind**
Fixes: https://github.com/go-gitea/gitea/issues/24850

Not sure how to do it for asian fonts only, so let's revert to previous value for now.

@lunny please try it, I'm not totally sure it solves what you see.

### Before
Screenshot 2023-05-22 at 10 34 10

### After
Screenshot 2023-05-22 at 10 34 04

### CODE REVIEW

**Comment by lunny** [comment]
Looks like only asian fonts affected. This current commits doesn't fix the problem.

图片

**Comment by silverwind** [comment]
So it must be one of these changes here:

https://github.com/go-gitea/gitea/commit/19993d8814e227ac0a52b73d36fdb03fbb143c3f#diff-9917ae0a071a9306b18b3b6e45b22df54cf8b7c4e6d8987d74dbfb1fd071d23b

A number of asian fonts went from 700 to 600. Is that the issue? Does it fix it when you set 700 on the affected elements?

**Comment by silverwind** [comment]
Maybe you can link a example on try.gitea.io so I can check there what the real issue is.

**Comment by silverwind** [comment]
Ah, I see the problem. Apparently it is the 700 -> 600 change in [font_i18n.css](https://github.com/go-gitea/gitea/commit/19993d8814e227ac0a52b73d36fdb03fbb143c3f#diff-9917ae0a071a9306b18b3b6e45b22df54cf8b7c4e6d8987d74dbfb1fd071d23b). When I set it back to 700, it actually renders less bold than before 😆. Will push a fix for you to try.

**Comment by silverwind** [comment]
@lunny try with https://github.com/go-gitea/gitea/pull/24855/commits/1c51ae63709d7bab41c3140eb24ed8817a738add applied. Pretty positive it should fix it for you.

File: web_src/css/repo.css [hunk 1]
```diff
@@ -627,7 +627,7 @@
   align-items: center;
   flex: 1;
   width: 100%;
-  font-weight: var(--font-weight-normal);
+  font-weight: var(--font-weight-light);
```
Comment (silverwind) [inline web_src/css/repo.css]:
```suggestion
  font-weight: var(--font-weight-normal);
```

**Comment by silverwind** [comment]
I removed the issue title change as it's been proven to be unrelated.

**Comment by silverwind** [comment]
@lunny does it work?

**Comment by lunny** [comment]
> @lunny does it work?

It doesn't work.

**Comment by silverwind** [comment]
> > @lunny does it work?
> 
> It doesn't work.

Can't be, the values are now exactly the same as before the PR. Please check in devtools which weight change actually gives you desirable result. I'm not sure what you actually want.

**Comment by HesterG** [comment]
This font here is using `var(--font-weight-normal)` as its `font-weight`, but ofr me, looks like 300,400,500 procude the same results, and 600 looks bolder.

Screen Shot 2023-05-23 at 19 04 14

Screen Shot 2023-05-23 at 19 00 44

Screen Shot 2023-05-23 at 19 00 52

Screen Shot 2023-05-23 at 19 01 02

**Review by lunny — APPROVED** [review]

**Review by HesterG — APPROVED** [review]

**Pull request merged**